generators/entity/templates/src/main/java/package/service/impl/_EntityServiceImpl.java

```diff
@@ -95,8 +95,14 @@ public void delete(<%= pkType %> id) {
      *  @param query the query of the search
      *  @return the list of entities
      */<% if (databaseType == 'sql') { %>
-    @Transactional(readOnly = true) <% } %>
-    public List<<%= instanceType %>> search(String query) {
-        <%- include('../../common/search_template', {viaService: viaService}); -%>
+    @Transactional(readOnly = true)<% } %>
```

Comment:
The stream template was the only one I could find where the code was the same. Is there anywhere else you can see?